packages/@aws-cdk/aws-apigateway/test/authorizers/integ.token-authorizer-iam-role.expected.json

```diff
@@ -36,7 +36,7 @@
       "Properties": {
         "Code": {
           "S3Bucket": {
-            "Ref": "AssetParameters6695c4a3dad80ddeef2797a1729306b7c136d67ce21d2187fdbc7bbad009993aS3Bucket115F9EA4"
```

Comment:
Curious, why did so many test expectations change? Shouldn’t this only affect source hashes that used “exclude”?

Reply:
So, `_hashField` [includes the header](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/assets/lib/fs/fingerprint.ts#L84) so it's updating even when `options.exclude` is not used.
We could do 
```
if (options.exclude){
  _hashField(hash, 'options.exclude', exclude.toString());
}
```
so it only changes when `exclude` was used. I was just following what was done previously with `options.extra` which was also optional (`_hashField(hash, 'options.extra', options.extraHash || '');`)

Reply:
Got it. Let’s do this conditionally to reduce the noise this might create (basically invalidating all assets everywhere)